### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by alpert**

### CODE REVIEW

File: src/docs/asciidoc/expert-zone/dag.adoc [hunk 1]
```diff
@@ -287,7 +287,7 @@ You should always set the local parallelism of the target vertex to 1,
 otherwise there will be idle processors that never get any items.
 
 On a local edge this policy doesn't make sense since simply setting the
-local parallelism of the target vertex to 1 constrains the local choice
+local parallelism of the target vertex to 1 constraints the local choice
```
Comment (viliam-durina) [inline src/docs/asciidoc/expert-zone/dag.adoc]: the original was correct

File: src/docs/asciidoc/jet-concepts.adoc [hunk 2]
```diff
@@ -398,7 +398,7 @@ exactly the event interval.
 image::session-window-1.png[Session window: single event,250,200,align="center"]
 
 
-A following event under the same key belongs to this window iff its
+A following event under the same key belongs to this window if its
```
Comment (viliam-durina) [inline src/docs/asciidoc/jet-concepts.adoc]: This is a common math/programming jargon: https://www.dictionary.com/browse/iff

File: src/docs/asciidoc/jet-concepts.adoc [hunk 3]
```diff
@@ -133,7 +133,7 @@ tokenizing processor:
 image::wordcount-tokenizer.png[Word-counting DAG with tokenizer vertex parallelized,600,170,align="center"]
 
 The input processor can now use all the available tokenizers as a pool
-and submit to any one whose queue has some room.
+and submit to anyone whose queue has some room.
```
Comment (viliam-durina) [inline src/docs/asciidoc/jet-concepts.adoc]: I think "submit to any whose..." would be best.

**Comment by alpert** [comment]
Made the changes accordingly.

File: src/docs/asciidoc/jet-concepts.adoc [hunk 4]
```diff
@@ -133,7 +133,7 @@ tokenizing processor:
 image::wordcount-tokenizer.png[Word-counting DAG with tokenizer vertex parallelized,600,170,align="center"]
 
 The input processor can now use all the available tokenizers as a pool
-and submit to any one whose queue has some room.
+and submit to any whose queue has some room.
```
Comment (viliam-durina) [inline src/docs/asciidoc/jet-concepts.adoc]: Discussed with a colleague, the original was actually correct. It stresses that the processor submits to one of the queues as opposed to any number of queues that have room.

**Comment by alpert** [comment]
Changed that too.

**Pull request merged**